pkg/kwokctl/runtime/kind/cluster.go

```diff
@@ -71,6 +73,38 @@ func getKindRuntimeExtraArgs(config *internalversion.KwokctlConfiguration, s str
 	return []internalversion.ExtraArgs{}
 }
 
+func getKindRuntimeExtaLogMounts(configPath string) ([]internalversion.Volume, error) {
+	objs, err := config.Load(context.Background(), configPath)
```

Comment:
```suggestion
func GetKindRuntimeExtaLogMounts(ctx context.Context, configPath string) ([]internalversion.Volume, error) {
	objs, err := config.Load(ctx, configPath)
```

The configuration is taken from the Context.
Let's move this function to `pkg/kwokctl/runtime` to allow docker runtime and Kind runtime to use it.